### DISCUSSION TITLE
PVE、OPNsense、Ubuntu Server设置小记 · TurboKre的博客 | Notes on Setting up PVE, OPNsense, and Ubuntu Server · TurboKre's Blog

### DISCUSSION THREAD

**Discussion by giscus** [ohmykreee/kreee-blog#16] [Comments]
# PVE、OPNsense、Ubuntu Server设置小记

A Kreee's Blog, powered by GitHub Pages, Hugo and Bilberry Premium Theme for Hugo.

https://blog.kre3.net/article/setup-pve-with-opnsense-ubuntu-notes/

**Comment by no1xiansen** [comment]
clash配置文件有吗

**Reply by ohmykreee** [reply]
可以参考 https://rxclc.club/index.php/archives/18/ 和 [官方示例](https://github.com/MetaCubeX/Clash.Meta/blob/Meta/docs/config.yaml) 以及 [官方文档](https://wiki.metacubex.one/config/)。

建议使用新的 mihomo 核心。
